sycl/include/sycl/stl_wrappers/cmath-fallback.h

```diff
@@ -0,0 +1,287 @@
+#ifndef __CMATH_FALLBACK_H__
+#define __CMATH_FALLBACK_H__
+
+#ifdef __SYCL_DEVICE_ONLY__
+
+#define __DPCPP_SYCL_DEVICE __attribute__((sycl_device_only, always_inline))
```

Comment:
Why do we need `sycl_device_only` attribute?
Can't the combination `ifdef __SYCL_DEVICE_ONLY__` + `sycl_device` attribute be used instead?